package.json

```diff
@@ -12,107 +12,104 @@
     "classnames": "~2.2.5",
```

Comment:
I think we should do the package upgrades in sync with the integreat webapp. So we stuble across issues only once.

Reply:
Yes, we can do it in sync from now on